common/beerocks/bwl/dwpal/sta_wlan_hal_dwpal.cpp

```diff
@@ -242,7 +242,42 @@ bool sta_wlan_hal_dwpal::connect(const std::string &ssid, const std::string &pas
 
 bool sta_wlan_hal_dwpal::associate(const sMacAddr &bssid, uint8_t channel)
 {
-    LOG(DEBUG) << "sta_wlan_hal_dwpal::associate() is not implemented!";
+    auto bssid_str = tlvf::mac_to_string(bssid);
+
+    LOG(DEBUG) << " Associate iface " << get_iface_name() << " to BSSID = " << bssid_str
+               << ", Channel = " << int(channel);
```

Comment:
As mentioned in #1424 this function isn't needed. It was wrong anyway because it didn't do any steering.

Reply:
This commit removed.